support-frontend/assets/components/secureTransactionIndicator/securePadlock.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="10" height="15" viewBox="0 0 10 15" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
+
```

Comment:
Would this still be necessary given that we have added `aria-hidden: "true"` which will take the svg out of the accessibility tree altogether?

Reply:
good question! It probably doesn't matter in that case but (and this is completely opinion) I'd add in the text just as best practice in following 'every image should have fallback text' in case it doesn't render

Reply:
Ah yes, good point!